app/src/androidTest/java/com/github/se/cyrcle/ui/report/AdminScreenTest.kt

```diff
@@ -156,4 +176,82 @@ class AdminScreenTest {
     composeTestRule.onNodeWithTag("ReportCard0").performClick()
     composeTestRule.onNodeWithTag("CheckReportsButton0").performClick()
   }
+
+  @Test
+  fun testAlertDialogAppearsOnMoreOptionsClick() {
+    composeTestRule.setContent {
+      AdminScreen(
+          navigationActions = navigationActions,
+          reportedObjectViewModel = reportedObjectViewModel,
+          parkingViewModel = parkingViewModel,
+          reviewViewModel = reviewViewModel)
+    }
+
+    // Expand the first report card
+    composeTestRule.onNodeWithTag("ReportCard0").performClick()
+    composeTestRule.waitForIdle()
+
+    // Click the More Options button
+    composeTestRule.onNodeWithTag("MoreOptionsButton0").performClick()
+    composeTestRule.waitForIdle()
+
+    // Assert dialog appears
+    composeTestRule.onNode(isDialog()).assertExists()
+
+    // Validate dialog content
+    composeTestRule.onNodeWithText("Details for this Reported Object:").assertExists()
+    composeTestRule.onNodeWithText("Loading parking details...").assertExists()
+
+    // Validate buttons
+    composeTestRule.onNodeWithText("Go To Parking").assertExists().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").assertExists().assertHasClickAction()
+
+    // Dismiss the dialog
+    composeTestRule.onNodeWithText("Return").performClick()
+    composeTestRule.waitForIdle()
+
+    // Ensure dialog is dismissed
+    composeTestRule.onNode(isDialog()).assertDoesNotExist()
+  }
+
+  @Test
+  fun testReportDetailsContentForParkingType() {
+    composeTestRule.setContent {
+      ReportDetailsContent(
+          objectType = ReportedObjectType.PARKING,
+          objectUID = parking1.uid,
+          userUID = parking1.owner,
+          reviewViewModel = reviewViewModel,
+          parkingViewModel = parkingViewModel)
+    }
+    composeTestRule
+        .onNodeWithTag("BulletPointParkingUID")
+        .assertExists()
+        .onChildren() // Focus on child nodes to inspect contents
+        .assertAny(hasText("Parking UID: Test_spot_1"))
+
+    composeTestRule
+        .onNodeWithTag("BulletPointParkingOwner")
+        .assertExists()
+        .onChildren()
+        .assertAny(hasText("Owner: user1"))
+
+    // Print the state of the root composable after validation
+    composeTestRule.onRoot().printToLog("ParkingDetailsTestRootAfterValidation")
```

Comment:
Is the `printToLog` necesary here?

Reply:
It's a remnant of my tests that I can remove, thanks. printToLogs are still a useful practice that I recommend when testing though!